server/src/server.ts

```diff
@@ -280,7 +280,7 @@ process.on("message", (msg: m.Message) => {
           hoverProvider: binaryExists,
           definitionProvider: binaryExists,
           completionProvider: binaryExists
-            ? { triggerCharacters: ["."] }
+            ? { triggerCharacters: [".", ">"] }
```

Comment:
Lol I guess triggerCharacters doesn't accept a compound `"->"`?

Reply:
It does not -- I too was hoping it would.